src/sql/workbench/contrib/notebook/browser/notebookViews/modal.component.ts

```diff
@@ -0,0 +1,21 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { Component, Input } from '@angular/core';
+
+@Component({
+	selector: 'views-modal-component',
```

Comment:
Would be best to be very clear here with the selector `notebook-views-modal-component`